parsing/intermediate/dialects/dialect1/src/test/java/org/apache/calcite/test/Dialect1ParserTest.java

```diff
@@ -4023,4 +4023,16 @@ private void checkExNotNull(SqlParserUtil.StringAndPos sap, Throwable thrown) {
         + "SELECT `BAR`";
     sql(sql).ok(expected);
   }
+
+  @Test public void testRenameProcedureTo() {
+    final String sql = "rename procedure foo to bar";
+    final String expected = "RENAME PROCEDURE `FOO` AS `BAR`";
+    sql(sql).ok(expected);
+  }
+
+  @Test public void testRenameProcedureAs() {
```

Comment:
nit: Let's either split out the testing of compound identifiers into a separate unit test or rename this unit test to reflect that it's exercising that as well (e.g. testRenameProcedureAsWithCompoundIdentifiers).